unitaryhack.md

```diff
@@ -30,8 +30,40 @@ If you're interested in claiming a cash prize, the following issue has a $100 bo
 
 [Create a quantum simulator in PyTorch](https://github.com/PennyLaneAI/pennylane/issues/1225)
 
-You can still claim swag prizes by contributing to these other unitaryHACK issues:
+You can claim swag prizes by contributing to these other unitaryHACK issues:
 
 [Add search bar for navigating tutorials](https://github.com/PennyLaneAI/pennylane/issues/1296)
 
+[Create a separate requirements file for the CI](https://github.com/PennyLaneAI/pennylane/issues/1292)
+
+[Change the order of the covariance matrix and the vector of means internally in default.gaussian](https://github.com/PennyLaneAI/pennylane/issues/1257)
+
+[Two operations named Interferometer](https://github.com/PennyLaneAI/pennylane/issues/1256)
+
+[Add a custom gate application for Toffoli in default.qubit](https://github.com/PennyLaneAI/pennylane/issues/1215)
```

Comment:
Oh, this one has been done, the issue should have autoclosed when the linked PR was merged 🤔